src/bin/genesis.rs

```diff
@@ -1,17 +1,37 @@
 //! A command-line executable for generating the chain's genesis block.
 
 extern crate atty;
+extern crate clap;
 extern crate serde_json;
 extern crate solana;
 
 use atty::{is, Stream};
+use clap::{App, Arg};
 use solana::entry_writer::EntryWriter;
 use solana::mint::Mint;
 use std::error;
 use std::io::{stdin, stdout, Read};
 use std::process::exit;
 
 fn main() -> Result<(), Box<error::Error>> {
+    let matches = App::new("solana-genesis")
+        .arg(
+            Arg::with_name("tokens")
+                .short("t")
+                .long("tokens")
+                .value_name("NUMBER")
+                .takes_value(true)
+                .required(true)
+                .help("Number of tokens with which to initialize mint"),
+        )
+        .get_matches();
+    let tokens: i64;
```

Comment:
Clippy won't like this.

Reply:
Weird, I checked the log. No complaint from clippy. It generally prefers:

```rust
let tokens = if { foo } else {bar};
```

Reply:
I imagine it's because of the `if let` construction in the following expression...?